app/Packages/TaskFighter/routes.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+use Illuminate\Http\Response;
+
+/*
+|--------------------------------------------------------------------------
+| API Routes
+|--------------------------------------------------------------------------
+|
+| Here is where you can register API routes for TaskFighter.
+|
+| Assumed:
+| - prefix = api/tasks
+| - namespace = TaskFighter\Controllers
+| - middleware = api
+|
+| @see RouteServiceProvider
+|
+*/
+
+
+Route::get('list/tick', function (Response $response) {
```

Comment:
Why is this not in a controller, why inline and why `DB::table` instead of `Task:all()`.